src/Inciter/AMR/edge.h

```diff
@@ -0,0 +1,80 @@
+#ifndef AMR_edge_t_h
+#define AMR_edge_t_h
+
+#include <iostream>
+#include <ostream>
+
+class edge_t {
+    using edge_ = std::pair<size_t, size_t>;
```

Comment:
Uh! I just realize that `edge_` is a type... Okay, for that I guess you have consistently used the `_t` suffix and I use capital first character. Then why don't you make it `edge_t`?